frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-patient-count/radiogram-card-content-patient-count.component.html

```diff
@@ -1,105 +1,133 @@
 <h5 class="card-title">Meldung über Patientenanzahlen</h5>
```

Comment:
You could use something like this to get rid of all of the async pipes. For multiple values just repeat the `ng-container` tag :)
```html
<ng-container *ngIf="totalPatientCount$ | async as totalPatientcount">
<!--- File content --->
</ng-container>
```